forem/forem
# Remove Vault Secrets page from admin interface

## What type of PR is this? (check all applicable)


## Description

Removes Vault Secrets page from Admin interface

## Related Tickets & Documents

- Closes #18487

## QA Instructions, Screenshots, Recordings

Check out the admin interface and ensure the Vault Secrets section is no longer available under `/admin/advanced/tools`

### UI accessibility concerns?

None

## Added/updated tests?


## [Forem core team only] How will this change be communicated?


## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
